xled/control.py

```diff
@@ -199,7 +199,7 @@ def get_mode(self):
         url = urljoin(self.base_url, "led/mode")
         response = self.session.get(url)
         app_response = ApplicationResponse(response)
-        required_keys = [u"code", u"mode", u"shop_mode"]
+        required_keys = [u"code", u"mode"]
```

Comment:
Maybe make this part of the followup commit?

Reply:
I can split the commit if you like. What do you want to do?

Reply:
In a followup commit you have improved those asserts to check only required part. Make this change part of that commit please.